test/test_priority_queue.jl

```diff
@@ -110,13 +110,13 @@ import Base.Order.Reverse
     @testset "PriorityQueueMethods" begin
         pq1 = PriorityQueue('a'=>1, 'b'=>2)
 
-        @testset "peek/get/dequeue!/get!" begin
+        @testset "peek/get/popfirst!/get!" begin
             @test peek(pq1) == ('a'=>1)
```

Comment:
Probably out of scope for this PR, but this use of `peek` should be replaces with `first`.
Which will
A) Nicely match to `popfirst`
B) Match `first` on a `Dict`
C) be consistent with `Base` using `peek` only for IO streams